perllib/FixMyStreet/App/Controller/Report/New.pm

```diff
@@ -1344,7 +1344,8 @@ sub set_report_extras : Private {
 
             my $value;
             if ($field->{datatype} eq 'multivaluelist') {
-                $value = $c->req->params->{$param_prefix . $field->{code}} // [];
+                my @params = $c->get_param_list($param_prefix . $field->{code}, 1);
```

Comment:
Do we want to allow comma based entry here? I assume we're not expecting that as a form of input, that's more for the around query strings passing in multiple status/categories and the like.
You could also do this as one line as `$value = [ $c->get_param_list($param_prefix . $field->{code}) ];`

Reply:
My bad - not needed